scripts/enter-env.sh

```diff
@@ -344,6 +344,34 @@ check_version ${CPU_ARCH}-elf-gcc $GCC_VERSION || return 1
 #
 #check_version ${CPU_ARCH}-elf-gdb $GDB_VERSION
 
+# Cmake Version
+###############
+if [ "$FIRMWARE" = "zephyr" ]; then
+	set -x
+	# Zephyr requires cmake 3.13.1 or higher
+	CMAKE_VERSION=$(cmake --version | head -n1 | cut -d' ' -f3)
```

Comment:
There is a check version function?

Reply:
`check_version` function verifies an exact version - here I check for version 3.13.1 **or higher**

I agree that it adds complexity, so the other solution I see is to force a 3.14.4 version in pip and simply check here for this version using `check_version`. What do you think @mithro?

Reply:
I fixed the cmake version to 3.14.4